webextensions/api/browserAction.json

```diff
@@ -243,6 +243,31 @@
               }
             }
           },
+          "string": {
+            "__compat": {
+              "description": "The <code>color</code> property of the <code>details</code> parameter can be set to a string.",
+              "support": {
+                "chrome": {
+                  "version_added": true
+                },
+                "edge": {
+                  "version_added": false
+                },
+                "firefox": {
+                  "version_added": true,
+                  "notes": [
+                    "Before Firefox 59, invalid color strings behaved as <code>null</code>."
+                  ]
+                },
+                "firefox_android": {
+                  "version_added": false
+                },
+                "opera": {
+                  "version_added": null
+                }
+              }
+            }
+          },
```

Comment:
I would not capture this as a separate subfeature. I'd capture it as 2 notes attached to `setBadgeBackgroundColor.__compat`:

* one for Edge, saying that passing a string as the value of `details.color` is not supported.
* one for Firefox, saying that before 59, passing an invalid string as the value of `details.color` is treated as `null`.

TBH I think it's a bit fuzzy sometimes, when to use notes and when to use subfeatures. So I'm not going to insist on this if you don't agree. I guess it's because it feels like passing a string value is a basic part of the API, not some extra new feature.